_posts/2021-02-19-understanding-NOPE.md

```diff
@@ -0,0 +1,177 @@
+
+There's something important hiding in this list. Look at the impact of the MM hedges in each situation. In some cases, the MM hedge moves _against_ the price action, providing a stabilizing effect. In other cases, the MM hedge moves _with_ the price action. Notice anything?
+
+**Important Thing #3: Selling options to MMs stabilizes price action. Buying options from MMs amplifies price action.**
+
+Hmm. So that means the aggregate balance between option buying and option selling is important. If they're in balance, the market moves normally. If option selling dominates, price movements are is dampened, unable to move as much as they might otherwise. And if option buying dominates...
+
+### Uh oh.
+
+Yeah. If more people are buying options than selling them, then MM hedging _amplifies price moves_.
+
+So, is this happening? Can we tell?
+
+The open interest for an option is the number of option contracts outstanding at each price. But that doesn't tell you whether the non-MM side of the transaction bought or sold. For that you have to watch the trades as they happen - or just buy the historical data.
+
+Of course, someone did that, so we know the answer:
+
+**Important Thing #4: Option buying outweighs option selling.**
+
+Which makes total sense from the perspective of any individual: buying options has limited downside risk, selling them has unlimited risk. Easy choice.
+
+In Lily-speak, we are now in an options degenerate market ([part 1](https://medium.com/swlh/options-degenerate-marketplaces-part-1-b0ddf1c96fa6), [part 2](https://medium.com/the-shadow/options-degenerate-marketplaces-part-2-57c9816c5977)). That is to say, a significant factor in price action is not how people value the asset but the forced moves that MMs make to hedge option trades.
+
+# What NOPE is measuring and how it does it
+
+Let's recap: we know option buying outweighs option selling, which means that price movements are amplified by MM hedging.
+
+Hold on though. A while back we asked how often MMs hedge price movements, and we said we didn't know. But we _can_ watch the option trades as they happen. So for each one, we can calculate the required hedge in the underlying, and then expect to see that trade come through. If we watch all option trades on all strikes on all expirations, we can build up a picture of "what MMs are going to do next".
+
+This is what NOPE is doing. In brief, the NOPE calculation is:
+
+1. Add up the deltas of all calls across all strikes across all expirations, weighted by volume; call it `all-call-deltas`
+2. Add up the deltas of all puts across all strikes across expirations, weighted by volume; call it `all-put-deltas`
+3. `NOPE = (all-put-deltas - all-call-deltas) / total-shares-traded`
```

Comment:
NOPE should be "Call delta + Put delta" since put delta is negative. Some people write this as "Call delta - put delta" since they like to take the absolute value of put delta.

Reply:
good catch, thanks - i went to the paper and still managed to misinterpret it!